python/graphstorm/gpartition/dist_partition_graph.py

```diff
@@ -147,6 +154,18 @@ def main():
 
         logging.info("DGL graph building took %f sec", part_end - time.time())
 
+    # Copy raw_id_mappings to dist_graph if they exist in the input
+    raw_id_mappings_path = os.path.join(args.input_path, "raw_id_mappings")
+
+    if os.path.exists(raw_id_mappings_path):
+        logging.info("Copying raw_id_mappings to dist_graph")
```

Comment:
Any reason we do the copy here?

Reply:
You here in particular or in why do we copy in general? We need to copy the mappings from their S3 location at the input (GSProcessing output) to where training will run, so that the ID remapping script can work. ID remapping assumes the `raw_id_mappings` directory exists at the same level as the part config JSON